Hymns/Home/HomeView.swift

```diff
@@ -3,32 +3,35 @@ import SwiftUI
 
 struct HomeView: View {
 
-    @State var searchText: String = ""
+    @State private var searchText: String = ""
+    @State private var searchActive: Bool = false
     @ObservedObject private var viewModel: HomeViewModel
 
     init(viewModel: HomeViewModel = Resolver.resolve()) {
         self.viewModel = viewModel
     }
 
     var body: some View {
-        VStack(alignment: .leading, spacing: 10) {
-            CustomTitle(title: "Look up any hymn")
-            searchBar
+        VStack(alignment: .leading) {
+            if !searchActive {
+                CustomTitle(title: "Look up any hymn")
+            }
+
+            SearchBar(
+                searchText: $searchText,
+                searchActive: $searchActive,
+                placeholderText: "Search by numbers or words")
+                .padding(.horizontal)
+
+            (searchActive ? Text("Recent searches") : Text("Recent hymns")).font(.caption).padding(.top).padding(.leading)
```

Comment:
Another thing that may just be my personal view. But to me swapping the text here just feels distracting to the experience. All of a sudden you see the text change and then you wonder what did that text say before? And so you go back to try and see what just changed. I would just leave them both to the more generic "Recent". Just less distracting and I don't think anyone would ever question what it means by recent.

Reply:
Yeah I thought it was a bit janky too. but Emily suggested making the text more specific to recent hymns and recent searches, since they are technically different things. I'm inclined to just keep it for now, until we get something she can look at and see her suggestions. It's easy to remove/change, so we don't lose out on anything if we keep it while if we change it now, we might forget about discussing it later

Reply:
Sure, thats fine. Yeah, I think what makes it so smooth is that it really does not feel like a view change when you go into search mode. But with the text change you kind of get the feeling like the view just changed more. Not as smooth. We can talk to Emily about it later.